db/migrate/20160127140802_add_digital_location_to_delivery_location.rb

```diff
@@ -0,0 +1,5 @@
+class AddDigitalLocationToDeliveryLocation < ActiveRecord::Migration
+  def change
+    add_column :locations_delivery_locations, :digital_location, :boolean
+  end
```

Comment:
Should we have the migration set true or false for existing locations, or would be better/easier to do this in the console later?